README.md

```diff
@@ -16,8 +16,8 @@ Please read [the tutorial](/docs/topics/lincheck-guide.md) that explains how to
 ##### Given talks:
  *  [Lincheck. Testing concurrent data structures in Java](https://www.youtube.com/watch?v=YAb7YoEd6mM) (Heisenbug 2019, RU)
  *  [Testing concurrent algorithms with Lincheck](https://nkoval.com/talks/#lincheck-joker-2019) (Joker 2019, RU)
```

Comment:
@ndkoval could you please provide the link for the corresponding video, I could not find it. And also don't forget to update the same reference in `Lincheck-guide.md`.

Reply:
https://www.youtube.com/watch?v=_0_HOnTSS0E

Reply:
I guess this talk refers to: Lock-free algorithms testing (Joker 2017, RU) 🙂

Reply:
Fixed